src/exchanges/gemini-client.js

```diff
@@ -5,14 +5,51 @@ const Level2Snapshot = require("../level2-snapshot");
 const Level2Update = require("../level2-update");
 const SmartWss = require("../smart-wss");
 
+class TickerCache {
+  constructor({ ask, bid, last, timestamp, market } = {}) {
```

Comment:
since we have to cache some data between updates I'm using this class to hold ticker data for each market